docs/DeveloperGuide.md

```diff
@@ -605,32 +648,104 @@ Similar to [<ins>UC07](#use-case-uc07-add-phone-number-to-contact) except duplic
 
     Use case ends.
 
-### **Use case: UC11 — Search for Contacts by Tag**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by tag.
+### **Use case: UC15 — Search for Contacts by Tag**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by tag.
 
-### **Use case: UC12 — Search for Contacts by Wedding**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by wedding.
+### **Use case: UC16 — Search for Contacts by Wedding**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by wedding.
 
-### **Use case: UC13 — Search for Contacts by Phone Number**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by phone number.
+### **Use case: UC17 — Search for Contacts by Phone Number**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by phone number.
 
-### **Use case: UC14 — Search for Contacts by Address**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by address.
+### **Use case: UC18 — Search for Contacts by Address**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by address.
 
-### **Use case: UC15 — Search for Contacts by Email Address**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for contacts by email address.
+### **Use case: UC19 — Search for Contacts by Email Address**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for contacts by email address.
 
 ---
 
-### **Use case: UC16 — Search for Wedding by Wedding Name**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for wedding by wedding name.
+### **Use case: UC20 — Search for Wedding by Wedding Name**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for wedding by wedding name.
 
-### **Use case: UC17 — Search for Task by Task Name**
-Similar to [<ins>UC10](#use-case-uc10-search-for-contacts-by-name) except searching for task by task name.
+### **Use case: UC21 — Search for Task by Task Name**
+Similar to [<ins>UC14](#use-case-uc14-search-for-contacts-by-name) except searching for task by task name.
 
 ---
 
-### **Use case: UC18 — Delete Contact**
+### **Use case: UC22 — Assign a Contact to one or more Weddings**
+
+**MSS**
+
+1. User <ins>lists all weddings (UC02)</ins>.
```

Comment:
Same comment as above, maybe needs extra step of listing contacts?

Reply:
yup, i'll add that thanks